src/usage_metrics/scripts/save_github_metrics.py

```diff
@@ -0,0 +1,135 @@
+"""This script pull github traffic metrics and saves them to a GC Bucket."""
+
+import json
+import logging
+import os
+import sys
+from dataclasses import dataclass
+from datetime import date
+
+import requests
+from google.cloud import storage
+from requests.exceptions import HTTPError
+
+logger = logging.getLogger()
+logging.basicConfig(level="INFO")
+
+
+@dataclass
+class Metric:
+    """Format metrics into folder names."""
+
+    name: str
+    folder: str
+
+
+TOKEN = os.getenv("API_TOKEN_GITHUB", "...")
+OWNER = "catalyst-cooperative"
+REPO = "pudl"
+BUCKET_NAME = "pudl-usage-metrics-archives.catalyst.coop"
+
+BIWEEKLY_METRICS = [
+    Metric("clones", "clones"),
+    Metric("popular/paths", "popular_paths"),
+    Metric("popular/referrers", "popular_referrers"),
+    Metric("views", "views"),
+]
+PERSISTENT_METRICS = [Metric("stargazers", "stargazers"), Metric("forks", "forks")]
```

Comment:
I think these probably don't need to be module-level variables and can instead be set within each function - the GH-auth related ones can go into `make_github_request` and the lists of metrics can be in `save_metrics`. Then it's clear that nobody else depends on these variables.